code/game/objects/structures/watercloset.dm

```diff
@@ -9,7 +9,7 @@
 	var/cistern = 0			//if the cistern bit is open
 	var/w_items = 0			//the combined w_class of all the items in the cistern
 	var/mob/living/swirlie = null	//the mob being given a swirlie
-	var/buildstacktype = /obj/item/stack/sheet/metal //they're metal now, shut up
+	var/buildstacktype = /obj/item/stack/sheet/iron //they're metal now, shut up
```

Comment:
I mean, they're iron now, surely? (Do we have to refactor meta comments when refactoring? Jesus.)

Reply:
```suggestion
	var/buildstacktype = /obj/item/stack/sheet/iron //they're iron now, shut up
```

Reply:
not even clear what it means anyway